ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/influxdb.j2

```diff
@@ -0,0 +1,43 @@
+{#
+ Copyright (c) 2023-2024 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# eos - influxdb #}
+{% if influxdb is arista.avd.defined %}
+!
+monitor telemetry influx
+{%     for destination in influxdb.destinations | arista.avd.natural_sort('name') %}
+   destination influxdb {{ destination.name }}
+{%         if destination.url is arista.avd.defined %}
+      url {{ destination.url }}
+{%         endif %}
+{%         if destination.database is arista.avd.defined %}
+      database name {{ destination.database }}
+{%         endif %}
+{%         if destination.data_retention_policy is arista.avd.defined %}
+      retention policy {{ destination.data_retention_policy }}
+{%         endif %}
+{%         if influxdb.vrf is arista.avd.defined %}
+      vrf {{ influxdb.vrf }}
+{%         endif %}
+{%         if destination.username is arista.avd.defined and destination.password is arista.avd.defined %}
+      username {{ destination.username }} password {{ destination.password_type | arista.avd.default("7") }} {{ destination.password | arista.avd.hide_passwords(hide_passwords) }}
+{%         endif %}
+{%     endfor %}
+{%     for socket in influxdb.source_sockets | arista.avd.natural_sort('name') %}
+   source socket {{ socket.name }}
+{%         if socket.url is arista.avd.defined %}
+      url {{ socket.url }}
+{%         endif %}
+{%         if socket.connection_limit is arista.avd.defined %}
+      connection limit {{ socket.connection_limit }}
+{%         endif %}
+{%     endfor %}
+{%     for tag in influxdb.tags | arista.avd.natural_sort('name') %}
+   tag global {{ tag.name }} {{ tag.value }}
+{%     endfor %}
+{%     if influxdb.source_standard_enabled is arista.avd.defined(false) %}
```

Comment:
The key should be `source_group_standard_disabled` so the logic should be inverted.

Reply:
Fixed.